src/subdataframe/subdataframe.jl

```diff
@@ -155,6 +155,8 @@ end
 ##
 ##############################################################################
 
+copy(sdf::SubDataFrame) = sdf[:]
```

Comment:
You could add `deepcopy` while you're at it.

Reply:
I think not if we want to be consistent with Base where: `deepcopy` of an array view is an array view, while `copy` of an array view is an array.

Reply:
Funny, I hadn't realized that. That array view method returns a view on a copied array... :-/